src/main/java/arona/task/Event.java

```diff
@@ -0,0 +1,57 @@
+package arona.task;
+
+import arona.exception.AronaInvalidDateException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a "Event" task,
+ * which is a task with a start time and end time attached.
+ *
+ * @author Maximilliano Utomo
+ */
+public class Event extends Task {
+    /**
+     * The start time of the task.
+     */
+    LocalDate startTime;
+    /**
+     * The end time of the task.
+     */
+    LocalDate endTime;
```

Comment:
Should this be `startDate` instead of `startTime`? Similarly, `endDate` instead of `endTime`?